spec/article_json/import/google_doc/html/image_parser_spec.rb

```diff
@@ -184,17 +184,19 @@
 
     context 'when the caption is `[image-link-to]`' do
       let(:text) { 'Original caption text' }
+      let(:second_text) { 'second text' }
       let(:caption_fragment) do
         "<p><span> [image-link-to: </span>" \
         "<span><a href=\"http://devex.com\">http://devex.com</a></span>" \
-        "<span>] #{text}</span></p>"
+        "<span>] #{text}</span><span>#{second_text}</p>"
```

Comment:
It probably doesn't make a difference for the text but the second `span` should probably be closed, right?

Reply:
Oh, good point, fixing it. thanks!

Reply:
Changed!